.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -8,3 +8,9 @@
 #### Links to related tickets (GitHub, Redmine, ...)
 
 *
+
+#### Documentation 
+
+* Brief summary of Documentation changed in this PR     
+* Links to Issues created for Documentation updates to be done (and note why they were not included in this PR)
```

Comment:
I don't understant the part about the `and note why they were not included in this PR`

Reply:
Yeah this feels pessimistic

Reply:
Hmm ok will re-word it.  it was meant to allow people space to note multi-PR dependencies, e.g. "Issue X was created for documentation to be updated when PRs Y and Z are both merged" 
What's a better phrasing to encourage people not to just make issues instead of including the documentation update in their PR?

Reply:
I don't think it's that common that people are deliberately punting documentation changes to issues instead of addressing them. I think people forget about documentation and the issue are opened post-merge.

I think maybe this section could just be a single bullet that says `Documentation files changed:` which will serve as a reminder to the author and the reviewers that documentation should be considered.

Reply:
Sure, agreed. Though I think it's equally important to list files which still need to be changed.

Reply:
For sure, I'm in favour of being even more strict though and getting all the documentation done in the same PR as the code changes. It should be an exception that documentation isn't updated at the same time. (It's definitely not what we're doing yet, but we can push to get there).

Reply:
Noted.  I just committed [2c674c4](https://github.com/aces/Loris/commits/2c674c) to simplify all this.